### PR TITLE
removed obsolete community release pipeline

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[link](https://examle.com)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* referenced pull request 1
* referenced pull request 2
* referenced pull request 2
etc. 

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: job-dsls/jobs/seed_jobs/kie_new_branches_seed_job.groovy [hunk 1]
```diff
@@ -53,7 +53,6 @@ job("${folderPath}/a-seed-job-${baseBranch}") {
                     "job-dsls/jobs/**/compile_downstream_build.groovy\n" +
                     "job-dsls/jobs/**/prod_projects_downstream_production.groovy \n" +
                     "job-dsls/jobs/**/dailyBuild_jdk11_pipeline.groovy\n" +
-                    "job-dsls/jobs/**/dailyBuild_jdk11_prod_pipeline.groovy\n" +
```
Comment (mareknovotny) [inline job-dsls/jobs/seed_jobs/kie_new_branches_seed_job.groovy:56]: this removal is not related to community ;)

Comment (mbiarnes) [reply job-dsls/jobs/seed_jobs/kie_new_branches_seed_job.groovy:56]: sure - it is not. But this is no t used for ages. we don't do a daily build for prod any more for community.

**Review by mareknovotny — APPROVED** [review]
